extensions/escape-game/standalone.mjs

```diff
@@ -161,31 +171,59 @@ class Standalone {
     if (typeof duration !== "number") {
       return false;
     }
-
-    ++this.doorOpenPressedCount;
-    await this.helper.broadcastCellSetUpdateToAllUser(
-      'update',
-      "world1",
-      CellSet.fromObject({
-        name: LAYER_DOOR.layerName,
-        cells: []
-      }),
-    );
-    setTimeout(async function(obj) {
-      if (--obj.doorOpenPressedCount == 0) {        
-        await obj.helper.broadcastCellSetUpdateToAllUser(
+    const mapName = "world1";
+    if (++this.doorOpenPressedCount === 1) {  
+      console.log("open!");
```

Comment:
Maybe `escape-room door is open due to e2s call.`?

Reply:
It is just my debug message ...... I think I will just remove it.